packages/nouns-webapp/src/config.ts

```diff
@@ -59,7 +59,7 @@ const app: Record<SupportedChains, AppConfig> = {
   [ChainId.Rinkeby]: {
     jsonRpcUri: createNetworkHttpUrl('rinkeby'),
     wsRpcUri: createNetworkWsUrl('rinkeby'),
-    subgraphApiUri: 'https://api.thegraph.com/subgraphs/name/nounsdao/nouns-subgraph-rinkeby-v4',
```

Comment:
TODO: reminder to revert this before merging to master

Reply:
Replace with `https://api.thegraph.com/subgraphs/name/nounsdao/nouns-subgraph-rinkeby-v5` instead, which targets the new contracts